libs/api-client/src/Network/Wire/Client/API/Conversation.hs

```diff
@@ -88,16 +91,17 @@ getConv cnv = do
         $ empty
     rsc = status200 :| [status404]
 
+-- | Create a conversation with the session user in it and any number of
+-- other users (possibly zero).
 createConv :: MonadSession m
```

Comment:
Here I'm generalizing `createConv` to make it work with arbitrary numbers of users, not just two-or-more